src/index.js

```diff
@@ -47,6 +51,10 @@ const gapic = Object.freeze({
  *
  * - `VideoIntelligenceServiceClient` - Reference to
  *   {@link v1beta2.VideoIntelligenceServiceClient}
```

Comment:
I believe this link needs to be updated to `v1`

Reply:
They should both be there.

Reply:
I misunderstood this. You are right. Fixed.